content/pt-br/docs/tasks/configure-pod-container/enforce-standards-namespace-labels.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Aplicando Padrões de Segurança de Pod com `Labels` em Namespace
+reviewers:
+- tallclair
+- liggitt
+content_type: task
+min-kubernetes-server-version: v1.22
+---
+
+Os namespaces podem ser rotulados para aplicar os [Padrões de segurança de pod]
+(/docs/concepts/security/pod-security-standards). As três políticas
```

Comment:
This link "Padrões de segurança de pod" is broken. I think it is because the newline after brackspaces.